package.json

```diff
@@ -128,7 +129,8 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-router-dom": "5.2.1",
-    "redux-promise-middleware": "6.1.2"
+    "redux-promise-middleware": "6.1.2",
+    "url": "0.11.0"
```

Comment:
This is not what I meant. My point was to go on the repo of the SDK that requires `url` as a dependency and add it there as a peer dep. Not here.

Reply:
ok. 
Will tool remove unused dependency `url` ?

Reply:
There is no automated tool to check deps, yet. Just remove the peer you added and leave the dependencies one

Reply:
ok. done